<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POPS Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-color: rgb(0, 0, 127);
}

#centralWidget{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QPushButton{
color: white;
background-color: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop:0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
}

QComboBox{
color: white;
background-color: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop:0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>225</y>
      <width>406</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="PartConL">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Part Con #/cc</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FlowRateL">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>35</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Rate cc/s</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="PressL">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pressure mbar</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TempCL">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>105</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp C</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="PartCon">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="FlowRate">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>35</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Press">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="TempC">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>105</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SkipSet">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Skip 0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skip 9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skip 99</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="BinsSet">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>160</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>16 Bins</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25 Bins</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 Bins</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 Bins</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Skip">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bins">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>New File</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="hPlot" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
